docs/index.md

```diff
@@ -1,12 +1,11 @@
-# Presidio - Data Protection and Anonymization API
+# **Presidio**: Data Protection and Anonymization SDK
 
-**Context aware, pluggable and customizable PII anonymization service for text and images.**
-
----
-
-## What is Presidio
-
-Presidio _(Origin from Latin praesidium ‘protection, garrison’)_ helps to ensure sensitive data is properly managed and governed. It provides fast **_identification_** and **_anonymization_** modules for private entities in text such as credit card numbers, names, locations, social security numbers, bitcoin wallets, US phone numbers, financial data and more.
+Presidio _(Origin from Latin praesidium ‘protection, garrison’)_ 
+helps to ensure sensitive data is properly managed and governed. 
+It provides fast **_identification_** and **_anonymization_** 
+modules for private entities in text and images such as 
+credit card numbers, names, locations, social security numbers, 
+bitcoin wallets, US phone numbers, financial data and more.
 
 ### Goals
```

Comment:
When you removed the ## What is Presidio - the rest of the Goals, Main Features, and Demo should be ## instead of ### - I'm adding suggestions for Goals and Demo - but since Main Features is not showing up in the Code to review it will have to be fixed manually

Reply:
Fixed, thanks!